doc/classes/MarginContainer.xml

```diff
@@ -4,7 +4,15 @@
 		Simple margin container.
 	</brief_description>
 	<description>
-		Simple margin container. Adds a left margin to anything contained.
+		Adds a top, left, bottom, and right margin to all [Control] nodes that are direct children of the container. To control the [MarginContainer]'s margin, use the [code]margin_*[/code] theme properties listed below.
+		[b]Note:[/b] Be careful, [Control] margin values are different than the constant margin values. If you want to change the custom margin values of the [MarginContainer] by code you should use the following examples:
+		[codeblock]
+		var margin_value = 100
```

Comment:
A small suggestion, but I'd still say that you don't need the `Note:` you can remove it in general. There again it's something that's nowhere to be found in the docs so it's not consistent with other people's work so far. Also, your notes are valuable docs, note side notes in my opinion.